tests/unittests/Foundation/NSThreadTests.m

```diff
@@ -57,15 +63,15 @@ - (void)countingloop {
     // THREAD_PRIORITY_HIGHEST and THREAD_PRIORITY_TIME_CRITICAL. Setting priority to half way between these makes it clear
     // what the priority is when conveyed back to the user.
 
-    EXPECT_EQ([myThread threadPriority], sExpectedHighThreadPriority);
+    EXPECT_LT([myThread threadPriority] - sExpectedHighThreadPriority, sMarginOfError);
```

Comment:
>[myThread threadPriority] - sExpectedHighThreadPriority [](start = 14, length = 55)

Should this be absolute value or are we always going to be higher than these values?